components/test_util/src/encryption.rs

```diff
@@ -32,6 +32,7 @@ pub fn new_file_security_config(dir: &tempfile::TempDir) -> EncryptionConfig {
         file_dictionary_rewrite_threshold: 100000,
         master_key: master_key_cfg.clone(),
         previous_master_key: master_key_cfg,
+        v2_directory_whitelist: Vec::new(),
```

Comment:
What will happen if there's one v2's folder missing from this list?

Reply:
(1) The keys cannot be found, and (2) this directory cannot be renamed (the rename will raise an error). (1) shouldn't really happen unless you change the whitelist half way. The plan is to hardcode it and forbid configuration.